packages/features/watchlist/lib/service/WatchlistOperationsService.ts

```diff
@@ -138,6 +140,26 @@ export abstract class WatchlistOperationsService {
       status: "BLOCKED",
     });
 
+    if (this.deps.scheduleBlockingService) {
```

Comment:
I think it shouldn't be an optional dependency, it is a critical part of this service.